app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/MyCameraImpl.kt

```diff
@@ -85,6 +81,9 @@ class MyCameraImpl(val context: Context) {
     }
 
     fun toggleSOS(): Boolean {
+        handleCameraSetup()
```

Comment:
Do we want maximum brightness when we are in `SOS` mode? 
On Android 13, currently, it would use the brightness set in normal flash mode

Reply:
This is the current behaviour